### PR TITLE
Replace samplePage with rosPage

### PR DESCRIPTION
**Pull request by kgaikwad**

### CODE REVIEW

File: sitemap.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:news="http://www.google.com/schemas/sitemap-news/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml" xmlns:mobile="http://www.google.com/schemas/sitemap-mobile/1.0" xmlns:image="http://www.google.com/schemas/sitemap-image/1.1" xmlns:video="http://www.google.com/schemas/sitemap-video/1.1">
 <url> <loc>https://cloud.redhat.com/insights</loc> </url>
 <url> <loc>https://cloud.redhat.com/ros</loc> </url>
-<url> <loc>https://cloud.redhat.com/sample</loc> </url>
+<url> <loc>https://cloud.redhat.com/ros</loc> </url>
 <url> <loc>https://cloud.redhat.com/oops</loc> </url>
 <url> <loc>https://cloud.redhat.com/no-permissions</loc> </url>
 </urlset>
```
Comment (apuntamb) [inline sitemap.xml:8]: just a small nitpick, new line at the end.

**Review by apuntamb — CHANGES_REQUESTED** [review]
For first round, looks good. Added an inline comment.

File: sitemap.xml (hunk 1 above)
Comment (kgaikwad) [reply sitemap.xml:8]: @apuntamb, 
With the help of `sitemap-builder` it auto creates sitemap.xml file.

**Review by apuntamb — APPROVED** [review]

**Comment by apuntamb** [comment]
Merged, thanks @kgaikwad!

**Pull request merged**